notebook/static/edit/js/editor.js

```diff
@@ -82,10 +82,10 @@ function(
             "Tab": "indentMore",
             "Shift-Tab" : "indentLess",
             "Cmd-/" : "toggleComment",
-            "Ctrl-/" : "toggleComment",
+            "Ctrl-/" : "toggleComment"
         },
         indentUnit: 4,
-        theme: "ipython",
+        theme: "default",
```

Comment:
Not sure why this change is part of this PR?

Reply:
Initially, the intent was to make the CodeMirror config (shortcuts, theme, etc.) consistent between code cells in the notebook and the file editor. The code cells use the "default" theme and the editor uses the "ipython" theme. Is this intentional?

Reply:
I don't know. I don't have any particular objection if you think it's the right thing to do, but I think it would have been better to do it as a separate PR so it could be discussed separately.

Git blame shows @minrk set it to use the IPython theme three years ago in 0efd335553afe8d84d4a16aed9fc37db643390f2 . Min, do you have any recollection whether there was a need for that?

Reply:
Oof, that's asking a lot of my foggy memory. Maybe it had to do with how magics were highlighted? Try highlighting some IPython-specific syntax and see if it works the same. If it looks fine, I've no objection.